registry/storage/driver/gdrive/gdrive_test.go

```diff
@@ -0,0 +1,135 @@
+package gdrive
```

Comment:
We have added the build tag, but circleci is failing , complaining that 
"no buildable Go source files in  ...."